src/server/node.rs

```diff
@@ -211,13 +211,23 @@ impl<C> Node<C>
         let store = self.store.clone();
         let ch = event_loop.channel();
 
+        let (tx, rx): (mpsc::Sender<i32>, mpsc::Receiver<i32>) = mpsc::channel();
```

Comment:
The type declaration acts like doc for readers/reviewers, although it's quite a simple use case.